src/image_transform.cpp

```diff
@@ -0,0 +1,245 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2024 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+
+#include "miopen/image_transform.hpp"
+#include "miopen/common.hpp"
+#include "miopen/execution_context.hpp"
+#include "miopen/find_solution.hpp"
+#include "miopen/image_transform/adjust_brightness/invoke_params.hpp"
+#include "miopen/image_transform/adjust_brightness/problem_description.hpp"
+#include "miopen/image_transform/adjust_contrast/invoke_params.hpp"
+#include "miopen/image_transform/adjust_contrast/problem_description.hpp"
+#include "miopen/image_transform/adjust_hue/invoke_params.hpp"
+#include "miopen/image_transform/adjust_hue/problem_description.hpp"
+#include "miopen/image_transform/adjust_saturation/invoke_params.hpp"
+#include "miopen/image_transform/adjust_saturation/problem_description.hpp"
+#include "miopen/image_transform/normalize/invoke_params.hpp"
+#include "miopen/image_transform/normalize/problem_description.hpp"
+#include "miopen/image_transform/solvers.hpp"
+#include "miopen/miopen.h"
+#include "miopen/names.hpp"
+#include "miopen/tensor.hpp"
+#include <cstddef>
+
+namespace miopen {
+
+miopenStatus_t miopenImageAdjustHue(Handle& handle,
```

Comment:
Please modify the function names in this file as follows:
miopen::miopenImageAdjustHue() to miopen::ImageAdjustHue()

Reply:
I've renamed the internal APIs